.eslintrc.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Do you have a plan to add our ESlint config? 😄 
You can put our config at the top:
```suggestion
  "extends": [
    "@nimblehq/eslint-config-nimble-react",
    "next/core-web-vitals"
  ]
```

---

Used our linter https://github.com/Lahphim/react-surveyor-challenge/pull/44/commits/fb342bdc39983cb2699c0cbced853cf8b9f73227